cernopendata/modules/fixtures/data/records/delphi-author-list-all.json

```diff
@@ -0,0 +1,4237 @@
+[
+  {
+    "accelerator": "CERN-LEP",
+    "authors": [
+        {
+          "affiliation": "LPNHE, IN2P3-CNRS, Univ. Paris VI et VII, Tour 33 (RdC), 4 place Jussieu, FR-75252 Paris Cedex 05, France",
+          "name": "Abdallah, J"
+        },
+        {
+          "affiliation": "LIP, IST, FCUL - Av. Elias Garcia, 14-$1^{o}$, PT-1000 Lisboa Codex, Portugal",
+          "name": "Abreu, P"
+        },
+        {
+          "affiliation": "DAPNIA - Service de Physique des Particules, CEA-Saclay, FR-91191 Gif-sur-Yvette Cedex, France",
+          "name": "Adami, F"
+        },
+        {
+          "affiliation": "CERN, CH-1211 Geneva 23, Switzerland; Institut fuer Hochenergiephysik, Oesterr. Akad. d. Wissensch., Nikolsdorfergasse 18, AT-1050 Vienna, Austria",
+          "name": "Adam, W"
+        },
+        {
+          "affiliation": "Dipartimento di Fisica Sperimentale, Universita di Torino; INFN, Via P. Giuria 1, IT-10125 Turin, Italy",
+          "name": "Della Riccia, B A"
```

Comment:
After the last fixes, the order of the authors is not in alphabetical order. Should that be fixed?